### DISCUSSION TITLE
D-dimensional polytope set operations

### DISCUSSION THREAD

**Discussion by YanBos** [CGAL/cgal#8384] [Q&A]
Hey there, 

I am currently working on a piece of software that requires the computation of intersections and unions of polytope sets similar as in the 2D Regularized Boolean Set Operations package. 
Does anybody have an idea on how to compute these operations (Combinatorial maps maybe?) or is there a plan to support them in the future? 

Any help would be appreciated :)

**Comment by gdamiand** [comment]
Hi Yan;

Combinatorial maps (and the geometrical layer Linear cell complex) can describe D-dimensional objects, and provide basic operations allowing modifications (mainly insertion and removal of cells). 

You can thus use it to develop intersection and union operations, but these operations are not (yet) implemented in CGAL (but this is probably a not very simple task to do...).
